src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -1685,5 +1685,28 @@ sap.ui.define([
 		return this.bSecureSearch;
 	};
 
+	/**
+	 * Determines the next link url should be used
+	 *
+	 * @param {string} sNextLinkUrl The next link url
+	 * @param {string} sPath The FHIR resource path
+	 * @param {object} mParameters Existing parameters
+	 * @returns {object} Next link object containing the url and parameters
+	 * @public
+	 * @since 2.3.2
+	 */
+	FHIRModel.prototype.getNextLink = function (sNextLinkUrl, sPath, mParameters) {
+		var sQueryParams = sNextLinkUrl.substring(sNextLinkUrl.indexOf("?") + 1, sNextLinkUrl.length);
+		var aParameter = sQueryParams ? sQueryParams.split("&") : [];
+		var aKeyValue;
+		for (var i = 0; i < aParameter.length; i++) {
+			aKeyValue = aParameter[i].split("=");
+			mParameters.urlParameters[aKeyValue[0]] = aKeyValue[1];
+		}
+		return { url: sPath, parameters: mParameters };
+	};
+
 	return FHIRModel;
-});
\ No newline at end of file
+});
+
+
```

Comment:
remove one empty line at the bottom of the file. One empty line is fine.

Reply:
done